packages/app-bridge/src/types/Emitter.ts

```diff
@@ -28,7 +29,7 @@ export type EmitterEvents = {
     };
 
     'AppBridge:TemplateAssetsUpdated': {
-        template?: 'documentPage' | 'library' | 'cover';
+        template?: ThemeTemplate;
```

Comment:
is this really nullable? Wouldn't you need a `Template` for `**Template**AssetsUpdated` action?

Reply:
and you are right in questioning it... fixed!